setup.py

```diff
@@ -39,7 +39,7 @@ def read(fname):
     keywords=['socorro', 'breakpad', 'crash', 'reporting', 'minidump',
               'stacktrace'],
     packages=find_packages(),
-    install_requires=[], # use peep -r requirements.txt instead
+    install_requires=[], # use pip -r requirements.txt instead
```

Comment:
Well, how else should you do it? 

In projects past I've made this a list by doing something like `install_requires=open('requirements.txt').read().splitlines()` but that's kinda silly because oftentimes `requirements.txt` has things you don't actually need in runtime.